app/src/main/java/com/sunlightlabs/congress/models/Bill.java

```diff
@@ -116,6 +116,14 @@ public static int sessionForYear(int year) {
             return 2;
     }
 
+    public static int yearFrom(int congress, int session) {
+        // first year of that Congress
+        int base = (((congress + 894) * 2) - 1);
+
+        // if it's session 2, up it by 1 year
+        return base + (session - 1);
+    }
+
```

Comment:
Convenience method, since the `year` is not a dedicated field on returned roll call information. I could also grab it from the `voted_at` (calculated) field, but it feels cleaner to do it this way.